src/docker/utils/prune.sh

```diff
@@ -132,11 +132,12 @@ docker rm -f $api 2>/dev/null
 docker volume rm -f $postgres-data
 
 if [ -n "$reset_localnet" ]; then
-	if [ -n "$CI" ]; then
-		sudo rm -rf $docker_dir/localnet/.aptos || true
-	else
-		rm -rf $docker_dir/localnet/.aptos || true
-	fi
+	# In order to avoid having to run `sudo rm -rf src/docker/localnet/.aptos`,
+	# we can bind-mount a volume one directory above `localnet/.aptos`, aka
+	# `localnet`, effectively giving the ephemeral container ownership of
+	# everything in that directory. Then we delete the localnet data, aka
+	# `.aptos`, from within the ephemeral container.
```

Comment:
```suggestion
	# In order to avoid having to run `sudo rm -rf src/docker/localnet/.aptos`,
	# we can run an ephemeral `docker` container that bind-mounts `localnet`.
	# Bind-mounting the parent of `.aptos` gives the container the right to
	# delete it.
```